multiversx_sdk_network_providers/proxy_network_provider_test.py

```diff
@@ -3,13 +3,11 @@
 from multiversx_sdk_core import Address
 
 from multiversx_sdk_network_providers.proxy_network_provider import (
-    ContractQuery,
-    ProxyNetworkProvider,
-)
+    ContractQuery, ProxyNetworkProvider)
 
 
 class TestProxy:
-    proxy = ProxyNetworkProvider("https://devnet-gateway.multiversx.com")
+    proxy = ProxyNetworkProvider("https://devnet-old-gateway.multiversx.com")
```

Comment:
this will only work temporarily. Why not use the new devnet?

Reply:
You are right. Will be adjusted in the near future.